### ISSUE TITLE
Bitbucket error: The details field cannot contain more than 2000 characters

### ISSUE THREAD

**Issue by kneefer** [saibotsivad/bpr-npm-audit#4]
Hi! Today we've started to see following issue in our pipelines. Maybe Bitbucket changed some constraints on their side?

```
+ npx bpr-npm-audit
npx: installed 1 in 1.254s
Could not push report to Bitbucket. 400 {"key": "report-service.general.bad-request", "message": "The details field cannot contain more than 2000 characters.", "arguments": {}}
```

**Comment by kneefer** [comment]
I wasn't aware there is already an open pull request for it: #3

**Comment by TheBrockEllis** [comment]
If you need a temporary fix until this gets merged in, I have a fork with the changes from that pull request that seem to work. We changed our pipeline to this:

```
    - npm run test
    - npm run build
    - npx fnoex/bpr-npm-audit#details-character-limit
```

**Comment by saibotsivad** [comment]
Fixed with #3 thanks to @TheBrockEllis for the quick fix 🎉
